src/jdk.accessibility/windows/man/jabswitch.md

```diff
@@ -1,5 +1,5 @@
 ---
-# Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2019, 2024, Oracle and/or its affiliates. All rights reserved.
 # DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
```

Comment:
I guess this was committed 3 weeks ago, so shouldn't it be just 2024?

Reply:
The file existed before, but in the Oracle-only closed repo.

Reply:
ok..LGTM